pandas/tests/io/test_stata.py

```diff
@@ -383,19 +383,15 @@ def test_encoding(self, version):
 
         # GH 4626, proper encoding handling
         raw = read_stata(self.dta_encoding)
-        with tm.assert_produces_warning(FutureWarning):
-            encoded = read_stata(self.dta_encoding, encoding="latin-1")
+        encoded = read_stata(self.dta_encoding)
         result = encoded.kreis1849[0]
 
         expected = raw.kreis1849[0]
         assert result == expected
         assert isinstance(result, str)
 
         with tm.ensure_clean() as path:
-            with tm.assert_produces_warning(FutureWarning):
-                encoded.to_stata(
-                    path, write_index=False, version=version, encoding="latin-1"
-                )
+            encoded.to_stata(path, write_index=False, version=version)
             reread_encoded = read_stata(path)
             tm.assert_frame_equal(encoded, reread_encoded)
```

Comment:
not sure if this is still testing anything useful

Reply:
Yea can probably just remove this altogether since it seems directed towards encoding

Reply:
there's also a the version kwarg.  @bashtage is this testing anything meaningful?

Reply:
It is testing correctness, but since encoding is a no-op already, I think isn't adding anything.  This said, I think it still uses some characters that aren't tested anywhere else.